justfile

```
@@ -49,3 +49,63 @@ check-nested-self:
 # Check not matching tags
 check-not-matching:
     cargo run --quiet -- show 7040402a6fe461068f5cf5296607c184d043a | less -R
+
+###################
+# HERE BE DRAGONS #
+###################
+launch-rb:
+    cargo run --quiet -- generate-stub "ruby" > tmp.rb
+    code tmp.rb
+    cargo run --quiet -- show
+    ls *.rb | entr -p cargo run --quiet -- run --command "ruby tmp.rb"
+
+launch-new-rb:
+    cargo run --quiet -- next
+    cargo run --quiet -- generate-stub "ruby" > tmp.rb
+    code tmp.rb
+    cargo run --quiet -- show
+    ls *.rb | entr -p cargo run --quiet -- run --command "ruby tmp.rb"
+
+launch-py:
+    code tmp.py
+    cargo run --quiet -- show
+    ls *.py | entr -p cargo run --quiet -- run --command "python3 tmp.py"
+
+# Requires Cargo.toml to look be like this:
+# [package]
+# name = "clash"
+# version = "0.1.0"
+# edition = "2021"
+# default-run = "clash"
+
+# [[bin]]
+# name = "tmp"
+# path = "tmp.rs"
+launch-rs:
+    code tmp.rs
+    cargo run --quiet -- show
+    ls *.rs | entr -p cargo run --quiet -- run \
+    --command "cargo run --bin tmp"
+
+launch-rs-debug:
+    code tmp.rs
+    cargo run --quiet -- show
+    ls *.rs | entr -p sh -c 'export RUST_BACKTRACE=1; cargo run --quiet -- run --ignore-failures --command "cargo run --bin tmp"'
+
+run:
+    cargo run -- run --command "ruby tmp.rb"
+
+# Test the stub generator with a random clash in ruby
+test-stub-rb:
+    cargo run -- next
+    cargo run -- generate-stub "ruby"
+
+# Test the stub generator with a random clash in rust
+test-stub-rs:
+    cargo run -- next
+    cargo run -- generate-stub "rust"
+
+# Test the stub generator with a random clash in C
+test-stub-c:
+    cargo run -- next
+    cargo run -- generate-stub "c"
```

you can reduce repetition by using parameters:
```suggestion
test-stub LANG:
    cargo run -- next
    cargo run -- generate-stub {{LANG}}
```

---

@daxida I think you missed something you are guilty on :)

---

Fixed in [7416270](https://github.com/Andriamanitra/clash/pull/31/commits/74162708894258608b8aa660b35dcc7d2705f710)